pkgs/development/libraries/irrlichtmt/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, zlib
+, libpng
+, libjpeg
+, libGL
+, libX11
+, libXxf86vm
+, withTouchSupport ? false
```

Comment:
Any reason why this is optional?

Reply:
Mostly to reflect upstream, which currently does not enable it by default. I think their concern is mostly not wanting to pull in another dependency, which is less of a concern here, but I wanted to err on the side of caution.

Reply:
Having to build this on a pinephone might not be ideal either 😃. But yeah, I can imagine that you would generally like to follow the advice of upstream. It's just that now, out of the box, touch support won't be working on a touch-enabled device and a (re)build would be needed.